spearhead/assets/sass/navigation.scss

```diff
@@ -21,3 +35,25 @@ $navigation-max-break-point: 'laptop-only';
 		}
 	}
 }
+
+.social-navigation > div {
+	ul li {
+		width: auto;
+	}
+}
+
+.primary-navigation-open .extra-navigation-wrapper {
+	background: var(--primary-nav--color-background);
+	padding: var(--global--spacing-unit) var(--global--spacing-horizontal);
+	visibility: visible;
+	opacity: 1;
+	top: 50px;
+	right: 0;
+	bottom: 30px; // Not sure why this div is offset by 30px
+	left: 0;
+}
+
+.admin-bar.primary-navigation-open .extra-navigation-wrapper {
+	top: calc(50px + var(--wpadmin-bar--height));
```

Comment:
It's something about this line that's triggering the animation bug. Removing it and the animation returns to the previous "slide up" effect. 

I thought it maybe had to do with the calc or use of a variable, but adding a static value here (e.g. `96px`) does not resolve the issue.